app/data/tests/test_views.py

```diff
@@ -156,6 +180,27 @@ def test_arbitrary_filters(self):
         response_data2 = json.loads(self.admin_client.get(url2).content)
         self.assertEqual(len(response_data2), 2)
 
+    def test_created_by(self):
+        url = '/api/records/?details_only=True'
```

Comment:
Would it be simpler to query the record details page like `'/api/records/{uuid}/?details_only=True'.format(uuid=record.uuid)`? That way we only get one record back and don't have to filter the response looking for our record below.